server/routerlicious/packages/lambdas/src/utils/messageGenerator.ts

```diff
@@ -48,3 +48,35 @@ export const createRoomLeaveMessage = (clientId: string): ISignalMessage => ({
 		content: clientId,
 	}),
 });
+
+export const IBroadcastSignalEventPayloadType = {
+	/**
+	 * Indicates that the data associated with an edit is or must be a `boolean`.
+	 */
+	RuntimeMessage: "RuntimeMessage",
+} as const;
+
+export const IRuntimeSignalMessageBody = {
```

Comment:
Wondering what the difference between this const IRuntimeSignalMessageBody  and the interface below are. How are they both used / are they both needed?

Reply:
Actually I like Alejandro's description suggestion below better than my own re-write to updating it to that: https://github.com/microsoft/FluidFramework/pull/16451#discussion_r1293989240